chart/kafka-connector/README.md

```diff
@@ -52,6 +52,7 @@ Additional kafka-connector options in `values.yaml`.
 | `gateway_url`      | The URL for the API gateway.                                                           | `http://gateway.openfaas:8080` |
 | `broker_host`      | location of the Kafka brokers.                                                         | `kafka`                        |
 | `print_response`   | Output the response of calling a function in the logs.                                 | `true`                         |
+| `fullnameOverride` | Override the name value used for the Connector Deployment object.                      | ``                             |
```

Comment:
I think this was meant to match the template which uses `connector.fullname`?

Reply:
Can we we use something else like `deploymentName`?

Reply:
sure

Reply:
If `fullnameOverride` is the standard, then this is my misunderstanding and we should revert to what you had.